packages/terra-base/src/Base.scss

```diff
@@ -31,3 +31,40 @@ body {
 *::after {
   box-sizing: inherit;
 }
+
+a {
+  background-color: transparent; /* Remove the gray background on active links in IE 10. */
+  color: var(--terra-link-color, #0065a3);
+  outline: none;
+  text-decoration: underline;
+}
+
+// stylelint-disable selector-no-qualifying-type
+a:visited,
+a.is-visited {
+  color: var(--terra-link-visted-color, #0065a3);
+}
+
+a:hover,
+a.is-hovered {
+  color: var(--terra-link-hover-color, #004c76);
+}
+
+a:focus,
+a.is-focused {
+  background: transparent;
+  border-radius: 0.2142rem;
+  box-shadow: 0 0 1px 3px rgba(76, 178, 233, 0.5), 0 0 7px 4px rgba(76, 178, 233, 0.35);
```

Comment:
This will need to be updated to allow theming. Update this to `var(--terra-link-focus-box-shadow, 0 0 1px 3px rgba(76, 178, 233, 0.5), 0 0 7px 4px rgba(76, 178, 233, 0.35));`

In the cerner-consumer-theme PR, you'll want to set this to: `--terra-link-focus-box-shadow: none`;

Reply:
[Updated](https://github.com/cerner/terra-core/pull/1171/commits/a9401c240610fba74ffaa99ce4e0cd4f5f1040b0#diff-1008bd5b69971980781839ebbefe2560R55).